src/tilings.jl

```diff
@@ -106,100 +111,100 @@ t|   |r
 cycletetrahedrongroup(q::Int, r::Int, s::Int, t::Int) = tetrahedrongroup([q, 2, t, r, 2, s])
 
 """
-    normalsubgroups(g::ReflectionGroup, maxindex::Int)
+    normalsubgroups(G::ReflectionGroup, maxindex::Int)
 
-Return all normal subgroups of `g` with index up to `maxindex`.
+Return all normal subgroups of `G` with index up to `maxindex`.
 """
-function normalsubgroups(g::ReflectionGroup, maxindex::Int)
-    gr = GAP.Globals.LowIndexNormalSubgroupsSearchForAll(g.group, maxindex)
-    lns = GAP.Globals.List(gr)
-    sbgrps = Vector{GapObj}()
-    len = GAP.Globals.Length(lns)
-    for i = 1:len
-        push!(sbgrps, GAP.Globals.Grp(lns[i]))
-    end
+function normalsubgroups(G::ReflectionGroup, maxindex::Integer)
+    lins_search = GAP.Globals.LowIndexNormalSubgroupsSearchForAll(__gapobj(G), maxindex)
+    sbgrps = GapObj[H for H in GAP.Globals.List(lins_search)]
     return sbgrps
-end
+end 
 
 """
-    fixedpointfree(subgroup::GapObj, g::ReflectionGroup)
+    fixedpointfree(subgroup::GapObj, G::ReflectionGroup)
 
-Return `true` if the `subgroup` of `g` is fixed-point free; otherwise `false`.
+Return `true` if the `subgroup` of `G` is fixed-point free; otherwise `false`.
 """
-function fixedpointfree(subgroup::GapObj, g::ReflectionGroup)
-    hom = GAP.Globals.NaturalHomomorphismByNormalSubgroup(g.group, subgroup)
-    fpf = true
-    G = GapObj(())
-    for i in 1:g.dimension
-        fpf = fpf && GAP.Globals.Image(hom, g.generators[i]) != G
-    end
-    i = 1
-    for pair in combinations(1:g.dimension, 2)
-        for j in 1:g.orders[i] - 1
-            fpf = fpf && GAP.Globals.Image(hom, (g.generators[pair[1]] * g.generators[pair[2]])^j) != G
+function fixedpointfree(subgroup::GapObj, G::ReflectionGroup)
+    hom = GAP.Globals.NaturalHomomorphismByNormalSubgroup(__gapobj(G), subgroup)
+    h(g) = GAP.Globals.Image(hom, g)
+    fixedpoint = any(isone ∘ h, gens(G))
+    fixedpoint && return false
+    
+    for g1 in gens(G)
+        for (g2, ord) in zip(gens(G), orders(G))
+            fixedpoint = any(isone ∘ h, (g1*g2^j for j in 1:ord - 1))
+            fixedpoint && return false
         end
-        i += 1
     end
-    return fpf
-end
+    return true
+end 
 
 """
-    orientable(subgroup::GapObj, g::ReflectionGroup)
+    orientable(subgroup::GapObj, G::ReflectionGroup)
 
-Return `true' if the `subgroup` of `g` is orientable; otherwise `false`.
+Return `true' if the `subgroup` of `G` is orientable; otherwise `false`.
 """
-function orientable(sbgrps::GapObj, g::ReflectionGroup)
-    gens = Vector{GapObj}()
-    for pair in combinations(1:g.dimension, 2)
-        push!(gens, g.generators[pair[1]] * g.generators[pair[2]])
-    end
-    gens = GapObj(gens)
-    gplus = GAP.Globals.Subgroup(g.group, gens)
-    return GAP.Globals.IsSubgroup(gplus, sbgrps)
+function orientable(sbgrps::GapObj, G::ReflectionGroup)
```

Comment:
Is there any reason to use `sbgrps` instead of `sbgrp`? The first to me implies a collection of subgroups but the function only deals with one

Reply:
I kept the name, but asked myself the same question

Reply:
We should change the name